bin/improver-temp-lapse-rate

```diff
@@ -94,14 +102,24 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
 
-    result = LapseRate(
-        max_height_diff=args.max_height_diff,
-        nbhood_radius=args.nbhood_radius,
-        max_lapse_rate=args.max_lapse_rate,
-        min_lapse_rate=args.min_lapse_rate).process(temperature_cube,
-                                                    orography_cube,
-                                                    land_sea_mask_cube)
-
+    if args.return_dalr:
+        result = temperature_cube.copy(
+            data=np.full_like(temperature_cube.data, U_DALR.points[0]))
+        result.rename('temperature_lapse_rate')
+        result.units = U_DALR.units
+        result.attributes['source'] = 'IMPROVER'
+        result.attributes['institution'] = 'Met Office'
+        add_history_attribute(result, ['IMPROVER'])
```

Comment:
I'm not sure that we're adding an IMPROVER history attribute anywhere else. It might be worth having a chat about the rationale for this.

Reply:
Again, this is part of ongoing discussions but as this has yet to be decided, I'm happy to strip out the history attribute instead if you feel strongly about this.